### DISCUSSION TITLE
Is there a purpose for taking other's ips?

### DISCUSSION THREAD

**Discussion by danielgallab** [theforumhelpers/theforumhelpers.github.io#270] [Q&A]
Even though I have an adblocker, most people don't or can't. I'm not sure why you need it, so my question is why.
> This data is not shared or sold with anybody besides site developers, who use it to improve the site based on usage statistics.

How do you use my ip to improve the site?

<sub><sup>this convo started in https://scratch.mit.edu/studios/3688309/comments/#comments-156452431</sup></sub>

**Comment by Accio1** [comment]
Please see my comments here https://scratch.mit.edu/studios/3688309/comments/#comments-156478825

In short, 
- IP addresses are collected by Google Analytics for geolocation purposes but never actually shown to site developers. 
- We use other information that Google Analytics provides (such as what devices users are visiting from) to improve the site.
    - This data is anonymized anyway, so there is no way to trace it back to you.
- All sites that you visit (including Scratch) have access to your IP Address if they want to access it.

**Reply by danielgallab** [reply]
> IP addresses are collected by Google Analytics for geolocation purposes but never actually shown to site developers.

Your privacy page says the opposite. May you please update it to say this?

**Reply by aidan-crockett** [reply]
#271 if you had checked the issues page

**Reply by Accio1** [reply]
> > IP addresses are collected by Google Analytics for geolocation purposes but never actually shown to site developers.
> 
> Your privacy page says the opposite. May you please update it to say this?

The Privacy Policy says that Google Analytics collects IP Addresses, which is correct. However, I agree that it is a bit confusing and users could misinterpret it, hence #271 ad @gosoccerboy5 mentioned above.
